backend/src/main/java/org/polsl/backend/controller/HardwareController.java

```diff
@@ -77,4 +77,15 @@ public HardwareController(HardwareService hardwareService) {
     hardwareService.deleteHardware(id);
     return ResponseEntity.ok(new ApiBasicResponse(true, "Usunięto sprzęt"));
   }
+
+  /**
+   * Endpoint obsługujący uzyskiwanie hardware'u o danym id.
+   *
+   * @param id ID wybranego sprzętu
+   * @return hardware o danym id
+   */
+  @GetMapping("/{id}")
+  public ResponseEntity<?> getHardware(@PathVariable(value = "id") Long id) {
+    return ResponseEntity.ok(hardwareService.getOneHardware(id));
+  }
```

Comment:
Tu oznaczenie jest jak najbardziej spoko, jedynie taka pierdoła - przenieśmy to obok drugiego GETa, to będzie łatwiej to znaleźć kiedyś😜